### PR TITLE
Comparing modifiedAt before and after migration and finish earlier wh…

### PR DESCRIPTION
**Pull request by PK85**
…en CIS has error and not called KEB at all

**Description**

Changes proposed in this pull request:

- Comparing modifiedAt before and after migration and finish earlier when CIS has error and not called KEB at all
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Review by ralikio — APPROVED** [review]

**Review by szwedm — DISMISSED** [review]
/lgtm

File: tools/cli/pkg/ers/model.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ type Instance struct {
 	Labels          string
 	LandscapeLabel  string
 	// TODO: Date comes in different format than specified
-	// ModifiedDate      date.Date //2021-10-04T112314.137Z
+	ModifiedDate   int //2021-10-04T112314.137Z
```
Comment (tillknuesting) [inline tools/cli/pkg/ers/model.go:29]:
```suggestion
	ModifiedDate   int64 //2021-10-04T112314.137Z
```
However I would suggest in general to use [time.Time](https://pkg.go.dev/time#Time) type that we have in the standard library for timestamps. However `2021-10-04T112314.137Z` looks malformed to me as to `RFC3339` it should be "2021-10-04T11:23:14.137Z"?

Example how to parse timestamp string:

```
package main

import (
	"fmt"
	"time"
)

type Instance struct {
	ModifiedDate time.Time
}

func main() {
	timeStampString := "2021-10-04T11:23:14.137Z"

	timeStamp, err := time.Parse(time.RFC3339, timeStampString)
	if err != nil{
		fmt.Println(err)
		return
	}

	instance := &Instance{}

	instance.ModifiedDate = timeStamp

	fmt.Println(instance.ModifiedDate)
} 
```

File: tools/cli/pkg/ers/command/migrate_all.go [hunk 2]
```diff
@@ -107,6 +107,7 @@ func (c *MigrationAllCommand) Run() error {
 
 	for _, instance := range instances {
```
Comment (tillknuesting) [inline tools/cli/pkg/ers/command/migrate_all.go:109]:
```suggestion
```

**Review by szwedm — DISMISSED** [review]
/lgtm

**Review by tillknuesting — DISMISSED** [review]

**Review by szwedm — APPROVED** [review]
/lgtm

**Pull request merged**